app/Http/Controllers/Auth/LoginController.php

```diff
@@ -27,13 +32,31 @@ class LoginController extends Controller
      */
     protected $redirectTo = '/';
 
+    private $employeeRepository;
+
     /**
      * Create a new controller instance.
      *
      * @return void
      */
-    public function __construct()
+    public function __construct(EmployeeRepository $employeeRepository)
     {
+        $this->employeeRepository = $employeeRepository;
         $this->middleware('guest', ['except' => 'logout']);
     }
-}
```

Comment:
This should be handled with a middleware and called in the constructor as it previously was instead of rewriting the login function. All custom middlewares are in the Middleware folder in `app/Http`.